packages/editor-ui/src/stores/users.ts

```diff
@@ -137,6 +146,9 @@ export const useUsersStore = defineStore(STORES.USERS, {
 		deleteUserById(userId: string): void {
 			Vue.delete(this.users, userId);
 		},
+		setCloudPLan(plan: CloudPlanData) {
+			this.cloudPlan = plan;
```

Comment:
just realized we are using the users store here.. maybe the usage store would be better? or else a new store?